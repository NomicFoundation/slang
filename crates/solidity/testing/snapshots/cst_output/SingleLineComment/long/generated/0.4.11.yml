# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ // Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris. Maecenas congue ligula ac quam viverra nec consectetur ante hendrerit. Donec et mollis dolor. Praesent et diam eget libero egestas mattis sit amet vitae augue. Nam tincidunt congue enim, ut porta lorem lacinia consectetur. │ 0..330

Errors: []

Tree:
  - SingleLineComment (Token): "// Lorem ipsum dolor sit amet, consectetur adip..." # 0..330
