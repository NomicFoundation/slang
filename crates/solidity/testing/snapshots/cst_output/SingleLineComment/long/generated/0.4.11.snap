# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

► Token(SingleLineComment) - (0..330)     │ // Lorem ipsum dolor sit amet, consectetur adip...
